harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -64,6 +65,8 @@ def test_require_global_batch_size(self) -> None:
         utils.ensure_requires_global_batch_size(pytorch_onevar_model.OneVarTrial, self.hparams)
 
     def test_onevar_single(self) -> None:
+        """Assert that the training loss and validation error decrease monotonically."""
+
```

Comment:
This is a duplicate of what the comment says.

If you want to put this here, probably delete the comment.

Reply:
Yeah, doc-string > comment